modules/data.atmosphere/R/download.Ameriflux.R

```diff
@@ -19,7 +19,7 @@ download.Ameriflux.site <- function(site_id) {
 ##' @param verbose should the function be very verbose
 ##' 
 ##' @author Josh Mantooth, Rob Kooper, Ankur Desai
-download.Ameriflux <- function(sitename, outfolder, start_date, end_date, overwrite=FALSE, verbose=FALSE) {
+download.Ameriflux <- function(sitename, outfolder, start_date, end_date, overwrite=FALSE, verbose=FALSE, ...) {
   # get start/end year code works on whole years only
   
   require(lubridate) #is this necessary?
```

Comment:
I'm sorry to keep griping about this–but it makes no sense to use `require()` in this way, given that the subsequent code depends on functions in `lubridate`. Use `library()` so an (informative) error occurs immediately!